dts/arm/st/l1/stm32l152Xc.dtsi

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 Abel Sensors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/l1/stm32l152.dtsi>
+
+/ {
+	sram0: memory@20000000 {
+		reg = <0x20000000 DT_SIZE_K(32)>;
+	};
+
+	soc {
+		flash-controller@40023c00 {
+			flash0: flash@8000000 {
+				reg = <0x08000000 DT_SIZE_K(256)>;
+			};
+		};
```

Comment:
```suggestion
		eeprom: eeprom@8080000{
			reg = <0x08080000 DT_SIZE_K(8)>;
		};
```

Reply:
Good catch, also should implement that in the stm32l151Xc dts files then